h2o-core/src/main/java/hex/PartialDependence.java

```diff
@@ -106,6 +106,7 @@ public void compute2() {
         Futures fs = new Futures();
         final double meanResponse[] = new double[colVals.length];
         final double stddevResponse[] = new double[colVals.length];
+        final double stdErrorofTheMeanResponse[] = new double[colVals.length];
```

Comment:
Maybe  stdErrorOfTheMeanResponse if it's camel case

Reply:
or maybe just SEMResponse

Reply:
yep, can make that change